code/__DEFINES/reagents.dm

```diff
@@ -184,12 +184,6 @@
 #define GIN 			"gin"
 #define ABSINTHE 			"absinthe"
 #define RUM 			"rum"
-#define TEQUILA 			"tequila"
-#define VERMOUTH 			"vermouth"
-#define WINE 			"wine"
-#define COGNAC 			"cognac"
-#define HOOCH 			"hooch"
-#define ALE 			"ale"
```

Comment:
No, as @ComicIronic said in that very same PR "These are duplicates because whichever moron made all the deadrum variants decided that they should use the same IDs as the original reagents, which needs a fix, actually."

/datum/reagent/ethanol/rum and /datum/reagent/ethanol/deadrum both have RUM as ID but that is a mistake.